CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.6.0 IN PROGRESS
 
 * Show modifying users' names instead of IDs. Refs UIROLES-71.
+* Use `Capabilities` components from `stripes-authorization-components` repository instead of local files. Refs UIROLES-86.
```

Comment:
Please mark it as a breaking change, since a new dependency in introduced

Reply:
new dependency doesn't mean it's breaking change

Reply:
@NikitaSedyx you're right, I thought it was a new peerDep

Reply:
@BogdanDenis please approve it